docs/core/tools/dotnet-workload-install.md

```diff
@@ -27,6 +27,13 @@ dotnet workload install -?|-h|--help
 
 The `dotnet workload install` command installs one or more *optional workloads*. Optional workloads can be installed on top of the .NET SDK to provide support for various application types, such as [.NET MAUI](/dotnet/maui/what-is-maui) and [Blazor WebAssembly AOT](https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-6-preview-4/#blazor-webassembly-ahead-of-time-aot-compilation).
 
+When the command is in workload-set update mode, workload versions installed:
```

Comment:
@baronfel more hyphenated entries...we should settle on what we want here.

Reply:
I was the one who suggested hyphenation when _workload set_ is used as an adjective (that's recommended by the MS Style Guide). I also think it makes it easier to read/parse, e.g. you could read it as "workload _set-update_ mode" without any hyphens.

Reply:
If this is meant to call-out to the `workload-set` CLI argument maybe we could wrap it in backticks so that it looks like code? 

Otherwise we could use a different framing that sidesteps the ambiguity: "When the .NET SDK is configured to use workload sets, ..."

Reply:
Yes we can format the CLI flag as code -- `workload-set` -- but what this was about is how it's used grammatically. For example:

Used as an adjective: ... not to a specific workload-set version.
Not used as an adjective: ...the preceding `install` command example pinned that workload set.

It's mostly used as an adjective, so there is a large number of hyphenated instances. 
It's not always going to be obvious to everyone how it's being used in any given instance, and not everyone who makes changes in this text is likely to be careful about when they hyphenate and when they don't ... so I think omitting the hyphen is probably a good choice here. Let me know if you still would like to do that and I'll de-hyphenate the adjectival forms that I hyphenated.

Reply:
@marcpopMSFT @baronfel 
I put all references to workload-set used as the CLI flag in code format. I'll merge the hyphenated and code-formatted text, and if you prefer fewer hyphens and code formatting, I'll open a follow-up PR to switch to non-hyphenated.